.github/workflows/npm-publish-cli.yml

```diff
@@ -21,12 +21,6 @@ jobs:
       - name: Build the CLI
         run: yarn workspace @storyblok/field-plugin-cli build
       - name: Publish the package to NPM
-        run: cd packages/cli && npm publish --access public --tag beta
-        env:
-          NODE_AUTH_TOKEN: ${{secrets.NPM_PUBLISH}}
-      - name: Tag latest to the published version
-        run: |
-          PACKAGE_VERSION=$(cd packages/cli && npm pkg get version --workspaces=false | tr -d \")
-          cd packages/cli && npm dist-tag add "@storyblok/field-plugin@${PACKAGE_VERSION}" latest
```

Comment:
We used to have this script to release a version to both `beta` and `latest` at the same time (to prevent people making mistake installing an old version), but let's remove this, and keep releasing to `rc` from now on until we hit the stable version.